### DISCUSSION TITLE
Does multiprocessing support only work when `minimal=True`?

### DISCUSSION THREAD

**Discussion by dclong** [Data-Centric-AI-Community/fg-data-profiling#961] [Q&A]
In my case, 
`pool_size=8` doesn't leverage multiprocessing unless I set `minimal=True`. 
Does multiprocessing support only work when `minimal=True`?

**Comment by sbrugman** [comment]
This is not the expected behaviour, `minimal` and `config.pool_size` should be unrelated ([relevant code](https://github.com/ydataai/pandas-profiling/blob/330669bba3045ce6f4d3fc2e18cc0482e8ade554/src/pandas_profiling/model/pandas/summary_pandas.py#L74)). If this issue still persists, please consider  [opening a bug report](https://github.com/ydataai/pandas-profiling/issues/new?assignees=&labels=bug&template=bug_report.md&title=).
